7.0/bookworm-slim/amd64/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM mcr.microsoft.com/dotnet/aspnet:7.0-bookworm-slim-amd64
```

Comment:
I thought 7.0 was still bullseye

Reply:
.NET 7 has bullseye and bookworm.

I've updated each distro to its latest version. I don't see a need for us to show examples of each distro version if there is nothing different other than the distro name.